addons/interact_menu/functions/fnc_userActions_addHouseActions.sqf

```diff
@@ -80,7 +80,7 @@ if ((vehicle ACE_player) != ACE_player) exitWith {};
             // systemChat format ["Add Actions for [%1] (count %2) @ %3", _typeOfHouse, (count _memPoints), diag_tickTime];
             {
                 private _helperPos = AGLtoASL (_houseBeingScaned modelToWorld (_houseBeingScaned selectionPosition _x));
-                private _helperObject = "ACE_LogicDummy" createVehicleLocal _helperPos;
+                private _helperObject = "ACE_LogicDummy" createVehicleLocal [0,0,0];
```

Comment:
Maybe add a comment so someone doesn't change it back while doing something else.